### ISSUE TITLE
Fresh data for you

### ISSUE THREAD

**Issue by OriginaIPenguin** [nilicule/StadiaGameDB#83]
Hi there! I've not used GitHub before, so apologies if I'm reaching out the wrong way. 

I've added a load more data to the database you may be interested in. I noticed someone asked for FPS in 1080p mode if the game is 30FPS in 4K. Luckily, I've already made headway collecting that info! 

My database has moved, so here's a link to all the different view. As usual, you have my permissions to use this data however you want (: 

https://airtable.com/shr32bmiOThVvSGar

**Comment by ja1984** [comment]
@OriginaIPenguin Your data is awesome!!

I have a different project inspired by this one ( https://github.com/ja1984/sogdb ).

Would it be OK for me to use your resolution information? Given full credit ofc!

**Comment by OriginaIPenguin** [comment]
Absolutely! I'm not part of Stadia Source anymore, so feel free to credit
me directly. 🙂
